build.gradle

```diff
@@ -54,7 +54,7 @@ ext {
 	hamcrestVersion = '2.2'
 	hibernateValidationVersion = '7.0.4.Final'
 	jacksonBomVersion = '2.13.3'
-	jaywayJsonPathVersion = '2.6.0'
+	jaywayJsonPathVersion = '2.7.0'
```

Comment:
**[UNRELATED]** A few of these can be updated but I thought I backed those changes out for a later proposal. I missed this one but will leave in for now.